frontend/components/board.js

```diff
@@ -39,128 +26,21 @@ module.exports = React.createClass({
   },
 
   render: function() {
+    var board = [];
+    var squareNum = 0;
+
+    for(var i=0; i<boardHeight; i++) {
```

Comment:
This looks good since we'll be able to maintain the size of the board with these vars instead of hardcoding the size.